xsd/siri_model/siri_time-v2.0.xsd

```diff
@@ -189,63 +197,129 @@ Rail transport, Roads and road transport
   </xsd:annotation>
  </xsd:element>
  <xsd:simpleType name="DayTypeEnumeration">
   <xsd:annotation>
-   <xsd:documentation>Values for Day Type TPEG pti_table 34</xsd:documentation>
+   <xsd:documentation>Values for TPEG Pti34 - DayType</xsd:documentation>
   </xsd:annotation>
   <xsd:restriction base="xsd:NMTOKEN">
-   <xsd:enumeration value="pti34_0"/>
-   <xsd:enumeration value="unknown"/>
-   <xsd:enumeration value="pti34_1"/>
-   <xsd:enumeration value="monday"/>
-   <xsd:enumeration value="pti34_2"/>
-   <xsd:enumeration value="tuesday"/>
-   <xsd:enumeration value="pti34_3"/>
-   <xsd:enumeration value="wednesday"/>
-   <xsd:enumeration value="pti34_4"/>
-   <xsd:enumeration value="thursday"/>
-   <xsd:enumeration value="pti34_5"/>
-   <xsd:enumeration value="friday"/>
-   <xsd:enumeration value="pti34_6"/>
-   <xsd:enumeration value="saturday"/>
-   <xsd:enumeration value="pti34_7"/>
-   <xsd:enumeration value="sunday"/>
-   <xsd:enumeration value="pti34_8"/>
-   <xsd:enumeration value="weekdays"/>
-   <xsd:enumeration value="pti34_9"/>
-   <xsd:enumeration value="weekends"/>
-   <xsd:enumeration value="pti34_10"/>
-   <xsd:enumeration value="holiday"/>
-   <xsd:enumeration value="pti34_11"/>
-   <xsd:enumeration value="publicHoliday"/>
-   <xsd:enumeration value="pti34_12"/>
-   <xsd:enumeration value="religiousHoliday"/>
-   <xsd:enumeration value="pti34_13"/>
-   <xsd:enumeration value="federalHoliday"/>
-   <xsd:enumeration value="pti34_14"/>
-   <xsd:enumeration value="regionalHoliday"/>
-   <xsd:enumeration value="pti34_15"/>
-   <xsd:enumeration value="nationalHoliday"/>
-   <xsd:enumeration value="pti34_16"/>
-   <xsd:enumeration value="mondayToFriday"/>
-   <xsd:enumeration value="pti34_17"/>
-   <xsd:enumeration value="mondayToSaturday"/>
-   <xsd:enumeration value="pti34_18"/>
-   <xsd:enumeration value="sundaysAndPublicHolidays"/>
-   <xsd:enumeration value="pti34_19"/>
-   <xsd:enumeration value="schoolDays"/>
-   <xsd:enumeration value="pti34_20"/>
-   <xsd:enumeration value="everyDay"/>
-   <xsd:enumeration value="pti34_255"/>
-   <xsd:enumeration value="undefinedDayType"/>
+   <xsd:enumeration value="unknown">
+    <xsd:annotation>
+     <xsd:documentation>TPEG Pti34_0, unknown</xsd:documentation>
+    </xsd:annotation>
+   </xsd:enumeration>
+   <xsd:enumeration value="monday">
+    <xsd:annotation>
+     <xsd:documentation>TPEG Pti34_1, Monday</xsd:documentation>
+    </xsd:annotation>
+   </xsd:enumeration>
+   <xsd:enumeration value="tuesday">
+    <xsd:annotation>
+     <xsd:documentation>TPEG Pti34_2, Tuesday</xsd:documentation>
+    </xsd:annotation>
+   </xsd:enumeration>
+   <xsd:enumeration value="wednesday">
+    <xsd:annotation>
+     <xsd:documentation>TPEG Pti34_3, Wednesday</xsd:documentation>
+    </xsd:annotation>
+   </xsd:enumeration>
+   <xsd:enumeration value="thursday">
+    <xsd:annotation>
+     <xsd:documentation>TPEG Pti34_4, Thusrday</xsd:documentation>
```

Comment:
<xsd:documentation>TPEG Pti34_4, Thursday</xsd:documentation>

Reply:
Corrected in next commit